src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* **Breaking change:** Fix space character encoding for baggage item values
```

Comment:
I suggest writing explicitly that space was encoded to `+` and now it is encoded to `%20`.

Reply:
changed in https://github.com/open-telemetry/opentelemetry-dotnet/pull/5303/commits/50ed9209edea6547f0b5af3716ac160da74a7718